setup/windows/README.md

```diff
@@ -49,6 +49,6 @@ $ ansible-playbook -i ../ansible-inventory ansible-playbook.yaml -vv
 ```
 
 The servers should logon automatically at boot and start the Jenkins slave, but there is no automatic restart procedure for crashes (for now).
-Hence, to start Jenkins after running `ansible-playbook`, the server must be restarted or the slave must be manually started with RDP.
```

Comment:
@gibfahn I like to restart the machine at the end to make sure that autologon and starting Jenkins work. Sometimes the machine does not logon automatically for reasons unknown and running that step again fixes it.

We could add the reboot at the end of the Ansible script. @kunalspathak can you add it to this PR and update the text here accordingly?